core-services/prow/02_config/_plugins.yaml

```diff
@@ -2858,6 +2858,8 @@ goose:
 heart:
   adorees:
   - openshift-merge-robot
+help:
+  help_guidelines_url: https://git.k8s.io/community/contributors/guide/help-wanted.md
```

Comment:
fyi this was added by `make update`

if it points to something I should have not done in my config/job changes I can't tell what that was

Reply:
@stevekuznetsov @alvaroaleman would you all know why my csi-driver-projected-resource repo changes in this PR would cause this change after running `make update` ?

i.e. expected or did I make a mistake in those changes ?  or was it a hiccup in `make update` and I should undo this one ?

thanks

Reply:
that doesn't seem expected, can you try if `make update` still wants that after a rebase?

Reply:
rebase alone seemed to do the trick @alvaroaleman thanks

the subsequent `make update` call did not change anything